inc/spbc-admin.php

```diff
@@ -250,6 +250,7 @@ function spbc_enqueue_scripts($hook)
         'is_main_site'                    => (int) is_main_site(),
         'img_path'                        => SPBC_PATH . '/images',
         'key_is_ok'                       => $spbc->key_is_ok,
+        'sfw_enabled'                     => $spbc->settings['sfw__enabled'],
```

Comment:
Please, rename this to `secfw_enabled`

Reply:
Done https://github.com/CleanTalk/security-malware-firewall/pull/218/commits/a6f1ee8fd40a59895577e14a371001e4e28bd032